packages/api/amplify_api/test/util.dart

```diff
@@ -122,7 +122,7 @@ class MockWebSocketConnection extends WebSocketConnection {
 
   MockWebSocketConnection(
       AWSApiConfig config, AmplifyAuthProviderRepository authProviderRepo)
-      : super(config, authProviderRepo, logger: AmplifyLogger());
+      : super(config, authProviderRepo, null, logger: AmplifyLogger());
```

Comment:
Generally want to avoid nullable positional parameters for this reason. Seeing a random null here is confusing to a reader. Even though it's just a test, it's a good example of why it's considered an anti-pattern.